types/react-hyperscript-helpers/index.d.ts

```diff
@@ -8,16 +8,18 @@ declare module 'react-hyperscript-helpers' {
 
   type Children<Props> = Props extends { children?: React.ReactNode } ? React.ReactNode[] : (Props extends { children?: unknown } ? [Props['children']] : never)
 
+  type WithKey<P> = P & { key?: string }
+
   interface HHelper {
```

Comment:
is a similar 'key' fix needed for HtmHelper type?

Reply:
No. Those accept keys now because the type for the first argument is coming from React's types (via `JSX.InstrinsicElements`) and includes key.

https://github.com/DataBiosphere/terra-ui/blob/0757db3536a400468093c3ea124650577c892d75/types/react-hyperscript-helpers/index.d.ts#L40

Added a check for it anyways in https://github.com/DataBiosphere/terra-ui/pull/3460/commits/b150b1de9eea08200054ca90434baa0eb82c9b8e.